docs/products/influxdb/getting-started.rst

```diff
@@ -0,0 +1,44 @@
+Getting started with Aiven for InfluxDB®
+########################################
+
+InfluxDB® is an open-source time series database developed by InfluxData. To get started with Aiven for InfluxDB, first select the "Create a new service" button.
+

```

Comment:
Please don't specify the image width, this needs to size with the context of the page. I'd also question whether this screenshot is really needed, or whether a user could locate the page they need from the text instructions. We will need to replace all screenshots soon so keeping these to a minimum is recommended.